weasel/cli/document.py

```diff
@@ -2,8 +2,9 @@
 
 from wasabi import MarkdownRenderer, msg
 
-from .._util import PROJECT_FILE, Arg, Opt, app, load_project_config
-from ..util import working_dir
+from weasel.utils import load_project_config, working_dir
```

Comment:
I am... The rule I set myself about this is that relative imports should not be used to access parent modules, since I find  it can quickly hinder legibility (although I'll admit one level is in general still quite readable).

I know it's not exactly the Explosion MO, let me know if you'd like to change it back.

Reply:
I just pushed a few commits, I think you'll prefer it that way.